corehq/apps/hqcase/templates/hqcase/explode_cases.html

```diff
@@ -1,76 +1,77 @@
-{% extends "hqwebapp/bootstrap3/base_section.html" %}
+{% extends "hqwebapp/bootstrap5/base_section.html" %}
 {% load hq_shared_tags %}
 
-{% requirejs_main 'hqcase/js/explode_cases' %}
+{% requirejs_main_b5 'hqcase/js/explode_cases' %}
 
 {% block page_content %}
   {% registerurl "users_select2_options" domain %}
 
-  <div id="explode">
+  <div id="explode" class="ko-template">
     <h1>Explode Cases 💣</h1>
     <h2><small>Use at your own risk 🙃</small></h2>
     <div class="row">
-      <div class="col-sm-12">
+      <div class="col-md-12">
         <p>This feature multiplies all the cases in a user's restore by <code>factor</code>.</p>
       </div>
     </div>
     <div class="row">
-      <div class="col-sm-8">
-        <div class="panel panel-danger">
-          <div class="panel-heading">Warning!</div>
-          <div class="panel-body">
-            <p>The case exploder will multiply all cases in the user's restore  by the explode factor entered below. So, for example, if you're exploding a test user with 20 cases, and set the factor to be 200, it will actually give you <strong>4,000</strong> cases, NOT 200. Be careful! For this reason, it's best to create a new test user with only one case to explode.</p>
+      <div class="col-md-8">
+        <div class="card border-warning mb-3">
+          <div class="card-header text-bg-warning">Warning!</div>
+          <div class="card-body">
+            The case exploder will multiply all cases in the user's restore  by the explode factor entered below. So, for example, if you're exploding a test user with 20 cases, and set the factor to be 200, it will actually give you <strong>4,000</strong> cases, NOT 200. Be careful! For this reason, it's best to create a new test user with only one case to explode.
           </div>
         </div>
-        <div class="panel panel-default">
-          <div class="panel-body">
+        <div class="card card-default mb-3">
```

Comment:
Nitpick: `card-default` isn't a class. The tool ought to be updated.

Reply:
https://github.com/dimagi/commcare-hq/pull/34669

Reply:
https://github.com/dimagi/commcare-hq/pull/34668/commits/0d931a373129e5a0958c3e5db6fdada45a3e1983